### PR TITLE
bugfix STOREID null for error: "Data did not match any of the schemas described in anyOf."

### PR DESCRIPTION
**Pull request by elisongit**
If customer subscribe newsletter as a guest, the variable $subscriber in the function beforeSubscribe (class Ebizmarts/MailChimp/Model/Plugin/Subscriber.php) has storeId null. This situation generate error "Data did not match any of the schemas described in anyOf." when module calls api for add subscribe in mailchimp.

We set storeId for resolve issue. For our company policy we modified also composer version because we editing a module third parts.

### CODE REVIEW

**Comment by gonzaloebiz** [comment]
Please, don't change the version.

File: composer.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   },
   "description": "Connect MailChimp with Magento",
   "type": "magento2-module",
-  "version": "1.0.21",
+  "version": "1.0.21.1",
```
Comment (centerax) [inline composer.json]: please see Gonzalo's comment. remove this change and the pr can be merged.

**Review by centerax — CHANGES_REQUESTED** [review]

**Comment by elisongit** [comment]
Thanks for reply, we add fourth digit in version composer for manage bugfix in third part modules.
Furthermore we installed Magento via composer and to download the updated version of module is necessary we modify the version in composer.json because composer checks version in composer.json to download module.

**Review by gonzaloebiz — CHANGES_REQUESTED** [review]
Please, don't change the version

**Comment by gonzaloebiz** [comment]
One month without news, so i close the PR

**Pull request closed without merging**